UnitsNet/UnitDefinitions/MolarMass.json

```diff
@@ -0,0 +1,42 @@
+﻿{
+  "Name": "MolarMass",
+  "BaseUnit": "KilogramPerMole",
+  "XmlDoc": "In chemistry, the molar mass M is a physical property defined as the mass of a given substance (chemical element or chemical compound) divided by the amount of substance.",
+  "Units": [
+    {
+      "SingularName": "GramPerMole",
+      "PluralName": "GramsPerMole",
+      "FromUnitToBaseFunc": "x/1e3",
+      "FromBaseToUnitFunc": "x*1e3",
+      "Prefixes": [ "Nano", "Micro", "Milli", "Centi", "Deci", "Deca", "Hecto", "Kilo" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "g/mol" ]
+        },
+        {
+          "Culture": "ru-RU",
+          "Abbreviations": [ "г/моль" ],
+          "AbbreviationsWithPrefixes": [ "нг/моль", "мкг/моль", "мг/моль", "сг/моль", "дг/моль", "даг/моль", "гг/моль", "кг/моль" ]
```

Comment:
Can we drop these prefixes and rely on the auto-generated prefixes, like the unit below?
It seems to me they only add the standard prefix letter here.

Reply:
Never mind, we can do that separately. No need to delay the PR over this.

Reply:
Yes, I thought those prefixes were necessary. But if it's not, let's drop them.

Reply:
Actually, it turns out only English is supported by the scripts. It should be trivial to extend the script to support other languages too as long as they simply prepend a certain string for each prefix. Russian seems to do this, but it is not a language I am familiar with.

Reply:
I am going to open an issue about this to remember later.